client/src/telethon/_impl/client/events/queries.py

```diff
@@ -43,6 +51,71 @@ def data(self) -> bytes:
         assert self._raw.data is not None
         return self._raw.data
 
+    @property
+    def via_inline(self) -> bool:
+        """
+        Whether the button was clicked in an inline message.
+
+        If it was, most likely bot is not in chat, and the :meth:`chat` property will return :data:`None`,
+        same for :meth:`get_message` method, however editing the message, using :meth:`message_id` property
+        and :meth:`answer` method will work.
```

Comment:
Reads a bit weird.

```suggestion
        If it was, it might indicate that the bot is not in chat.
        If this is the case, both the :meth:`chat` and :meth:`get_message` will return :data:`None`.
```

I think the remark on edit / answer can be omitted. Mentioning which methods won't work should be enough.